README.md

```diff
@@ -7,30 +7,45 @@
 What is webpki?
 ==================
 
-webpki is a library that validates Web PKI (TLS/SSL) certificates. webpki
-is designed to provide a **full** implementation of the client side of the
-**Web PKI** to a diverse range of applications and devices,
-including embedded (IoT) applications, mobile apps, desktop applications, and
-server infrastructure. webpki is intended to not only be the best
-implementation of the Web PKI, but to also *precisely define* what the Web PKI
-is.
+webpki is a library that validates Web PKI (TLS/SSL) certificates. It's
```

Comment:
So I feel like there's three sections here:

* "What is webpki" -- though this is so generic I'd probably just get rid of this header. I think this includes both your initial sentences here, the "written in Rust and uses *ring*" part, and also a blurb about the fork history.
* Features -- the bullet list of things included in the scope
* "What can't webpki do" -- I typically call this section Limitations

Reply:
I like this arrangement, thanks :-) Updated to adopt it.